webapp/src/app/pages/components/knowledge-graph/result-view/result-view.component.css

```
@@ -0,0 +1,87 @@
+
+:host ::ng-deep #cro_sorting .p-panel-title {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)